### PR TITLE
:sparkles: Improved the look of the pages slightly

### PR DESCRIPTION
**Pull request by DanC123**
Just centered some of the text and changed some colours around, nothing too drastic.

### CODE REVIEW

File: WhereIsMyColleague.Web/Views/Shared/_Layout.cshtml [hunk 1]
```diff
@@ -1,36 +1,37 @@
-﻿<!DOCTYPE html>
+﻿@using System
```
Comment (st3v3nhunt) [inline WhereIsMyColleague.Web/Views/Shared/_Layout.cshtml]: I think you can remove this?

**Comment by st3v3nhunt** [comment]
Have the css changes gone into `bootstrap.css`?

**Comment by DanC123** [comment]
Are you sure? VS was complaining about it not being there for the DateTime in the footer.

The css changes have gone into `bootstrap.css` yeah.

File: WhereIsMyColleague.Web/Views/Shared/_Layout.cshtml [hunk 2]
```diff
@@ -1,36 +1,37 @@
...
-      </footer>
-    </div>
-  </body>
-</html>
+  </div>
+  <div class="container body-content">
+    @RenderBody()
+    <hr />
+    <footer>
+      <p>&copy; @DateTime.Now.Year </p>
```
Comment (st3v3nhunt) [inline WhereIsMyColleague.Web/Views/Shared/_Layout.cshtml]: Can the trailing space after the Year be removed?

**Comment by st3v3nhunt** [comment]
@DanC123 re using statement in the view, it works OK for me.

**Comment by st3v3nhunt** [comment]
The site specific styling should go into `site.css`. `bootstrap.css` should be seen as an external artifact that we don't want to go messing with. At least most of the time.

**Comment by st3v3nhunt** [comment]
@DanC123, i'm going to hold off merging this as I think @jbroodie might have a :broken_heart: if I were to merge these changes in front of his.
I don't really fancy getting a :facepunch:

**Comment by DanC123** [comment]
Thats fine with me, I only did a little bit of editing so it looked a bit nicer for when Sean see's it. Hope you noticed I got the emoji right this time!

**Comment by st3v3nhunt** [comment]
I'm waiting for an updated avatar...

**Comment by DanC123** [comment]
If you mean my GitHub avatar I might have to get Paint open again. I'm thinking something that shows my Red Team pride, not that you'd know anything about that seeing as you just left us.

**Comment by st3v3nhunt** [comment]
@DanC123 Can you rebase this against the latest changes in master please?

**Pull request merged**